KestrelMock.Tests/DynamicMocks/response_after_dynamic_mock_creation___given_null_settings.cs

```diff
@@ -0,0 +1,23 @@
+﻿using FluentAssertions;
+using KestrelMockServer.Services;
+using Xunit;
+
+namespace KestrelMock.Tests.DynamicMocks;
+
+public class response_after_dynamic_mock_creation___given_null_settings
+{
+    private readonly DynamicMockAddedResponse response;
+
+    public response_after_dynamic_mock_creation___given_null_settings()
+    {
+        this.response = DynamicMockAddedResponse.Create(null);
```

Comment:
This was one of the unhandled issues the other day ... but added a few tests over the rest of the behaviour in this area.